app/src/ui/preferences/git.tsx

```diff
@@ -151,7 +165,14 @@ export class Git extends React.Component<IGitProps, IGitState> {
         )}
 
         <p className="git-settings-description">
-          These preferences will edit your global Git config.
+          These preferences will{' '}
+          {this.props.selectedExternalEditor
+            ? (
```

Comment:
```suggestion
          {this.props.selectedExternalEditor && this.state.globalGitConfigPath
            ? (
```

This will prevent having a dead link if for some reason the globalGitConfigPath can't be found. 

Warning: Accepting this suggestion may break the linter formatting check. If so, just need to run `yarn lint:fix` locally.

Reply:
Implemented 👍🏼